app/helpers/ncr/work_orders_helper.rb

```diff
@@ -1,9 +1,8 @@
 module Ncr
   module WorkOrdersHelper
     def approver_options
-      # @todo should this list be limited by client/something else?
       # @todo is there a better order? maybe by current_user's use?
-      User.order(:email_address).pluck(:email_address)
+      User.where(client_slug: 'ncr').order(:email_address).pluck(:email_address)
```

Comment:
I'm fine with keeping this method as-is, since the query is relatively simple.

> Per this conversation with Aidan, however, I'm trying to keep things out of the model that are just view-based extensions

That specific case was that it was returning a placeholder string (essentially, `"user not found"`) when the query didn't return anything, which definitely feels like it falls under the realm of Helpers. I'm kind of indifferent about where queries [that return data] live, even if they are only being used for views.

> Another option would be to create a query object

Very :+1: on query objects in general! Especially for complex queries, or very purpose-specific ones. We have some under [`lib/query/`](https://github.com/18F/C2/tree/master/lib/query) already.